addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -145,6 +145,26 @@ $bronze: #eea91e;
     .forum_answer img, .question-block img {
         max-width: 100%
     }
+
+    .o_wforum_index_entry_tags {
+        @for $size from 1 through length($o-colors) {
+            .o_tag_color_#{$size - 1} {
+                $background-color: white;
+                // no color selected
+                @if $size == 1 {
+                    color: $gray-600;
+                    background-color: $background-color;
+                } @else {
+                    color: white;
+                    background-color: nth($o-colors, $size);
+                }
+                &:hover {
+                    color: color-yiq($background-color);
+                    background-color: darken($background-color, 10%);
+                }
+            }
+        }
```

Comment:
Ok, I checked and you c/c this from another app without thinking about it. ***Code is to be shared.*** In this case, you are not only duplicating code, you are duplicating CSS rules which is bad for the loading, for efficiency, ... and also the original rules were bad anyway, so you are duplicating something bad.

Remove everything related to this from the PR and we'll check later how to share and improve this code.